src/sentry/static/sentry/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -125,6 +123,8 @@ class ActionsPanel extends React.PureComponent<Props> {
           : null,
       targetIdentifier: '',
       integrationId: actionConfig.integrationId,
+      inputType: actionConfig.inputType || null,
```

Comment:
@ceorourke why do we need the default to `null` here?

Reply:
I can look into this more but email seems to pass null